src/components/Button/Button.scss

```diff
@@ -12,24 +12,9 @@ $block: '.#{variables.$ns}button';
     --_--border-color: currentColor;
     --_--focus-outline-color: var(--g-color-line-focus);
     --_--focus-outline-offset: 0;
-    --_--icon-space: calc(
-        var(--g-button-padding, var(--_--padding)) + var(--g-button-icon-size, var(--_--icon-size)) +
-            var(--g-button-icon-offset, var(--_--icon-offset))
-    );
-    --_--icon-position: calc(
-        var(--g-button-padding, var(--_--padding)) -
-            (
-                var(--g-button-height, var(--_--height)) - var(
-                        --g-button-icon-size,
-                        var(--_--icon-size)
-                    )
-            ) / 2
-    );
     --_--font-size: var(--g-text-body-1-font-size);
 
     @include mixins.button-reset();
-    display: inline-block;
-    position: relative;
```

Comment:
There are children elements with `absolute` position (:before, :after), this prop can't be omitted

Reply:
fixed 79ccb7a622dc73a312669206b7b58c3ab85dc9e9